### PR TITLE
Use date and time inputs, and polyfill

### PR DESCRIPTION
**Pull request by spacetag**
Address #68

### CODE REVIEW

File: code_court/courthouse/templates/base.html [hunk 1]
```diff
@@ -10,9 +10,13 @@
 
         <title>code_court</title>
 
-        <script src="{{ url_for('main.send_static', path='js/jquery-3.1.1.min.js') }}"></script>
+        <script src="{{ url_for('main.send_static', path='js/jquery-2.2.4.min.js') }}"></script>
```
Comment (BenDoan) [inline code_court/courthouse/templates/base.html]: Why are we downgrading the jquery version?

**Comment by spacetag** [comment]
Webshim Polyfill doesn't support jQuery 3.x :-(

https://github.com/aFarkas/webshim

File: code_court/courthouse/templates/base.html (hunk 1 above)
Comment (spacetag) [inline code_court/courthouse/templates/base.html]: Webshim Polyfill doesn't support jQuery 3.x :-(

https://github.com/aFarkas/webshim

**Comment by spacetag** [comment]
Fixes #115

**Pull request merged**